src/storage.rs

```diff
@@ -28,9 +28,15 @@ pub struct Data {
     pub value: Value,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct DataCollection {
+    pub results: Vec<Value>
```

Comment:
It should be, yes.  Couldn't get that to deserialize though, will take another look before merging this.

Reply:
Hmm weird. Since Data has `#[derive(Seralize, Deserialize)]` this should be a non-issue. It's possible Vec is not serializable without a manual implementation.

Reply:
Hmm weird. Since Data has `#[derive(Seralize, Deserialize)]` this should be a non-issue. It's possible Vec is not serializable without a manual implementation.

Reply:
Fixed this, not really sure what wasn't before but works fine now